stripe2qbo/api/tests/test_app.py

```diff
@@ -0,0 +1,33 @@
+from fastapi.testclient import TestClient
+
+from stripe2qbo.api.app import app
+
+client = TestClient(app)
+
+
+def test_index():
+    response = client.get("/")
+    assert response.status_code == 200
+    assert "text/html" in response.headers["content-type"]
+
+def test_user_id_unauthorized():
```

Comment:
There should be 2 lines btw each function. Can you make sure `pre-commit` is running? It shouldn't let you commit until the formatters/linters are appeased

https://pre-commit.com/

Reply:
Added pre-commit